core/management/commands/new_event.py

```diff
@@ -6,13 +6,11 @@
 from django.template.loader import render_to_string
 from slacker import Error as SlackerError
 
-from core.command_helpers import gather_event_date_from_prompt
-from core.default_eventpage_content import (get_default_eventpage_data,
-                                            get_default_menu)
-from core.forms import AddOrganizerForm
-from core.models import Event, EventPageContent, EventPageMenu
-from core.slack_client import slack
-from core.utils import get_coordinates_for_city
+from ...command_helpers import gather_event_date_from_prompt
+from ...forms import AddOrganizerForm, EventForm
+from ...models import Event
+from ...slack_client import slack
+from ...utils import get_coordinates_for_city
```

Comment:
I'm not feeling too strongly about it though, so feel free to ignore that comment as well.